splink/profile_data.py

```diff
@@ -230,19 +233,28 @@ def profile_columns(linker, column_expressions, top_n=10, bottom_n=10):
         percentile_rows = [
             p for p in percentile_rows_all if p["group_name"] == _group_name(expression)
         ]
-        percentile_rows = _add_100_percentile_to_df_percentiles(percentile_rows)
-        top_n_rows = [
-            p for p in top_n_rows_all if p["group_name"] == _group_name(expression)
-        ]
-        bottom_n_rows = [
-            p for p in bottom_n_rows_all if p["group_name"] == _group_name(expression)
-        ]
-        # remove concat blank from expression title
-        expression = expression.replace(", ' '", "")
-        inner_chart = _get_inner_chart_spec_freq(
-            percentile_rows, top_n_rows, bottom_n_rows, expression
-        )
-        inner_charts.append(inner_chart)
+        if percentile_rows == []:
```

Comment:
Can you add a test for this?

You can shove it in [here](https://github.com/moj-analytical-services/splink/blob/master/tests/test_profile_data.py).

Reply:
Have added test for the warning message it produces to test. Do you feel percentile rows needs testing specifically?